src/nvim/options.lua

```diff
@@ -8432,19 +8432,19 @@ return {
         Otherwise this option does not always reflect the current syntax (the
         b:current_syntax variable does).
         This option is most useful in a modeline, for a file which syntax is
-        not automatically recognized.  Example, in an IDL file: >
+        not automatically recognized.  Example, in an IDL file: >c
```

Comment:
```suggestion
        not automatically recognized.  Example, in an IDL file: >idl
```

Reply:
We don't have IDL injections; C is the closest (and will give correct highlighting, since it's only a C-style comment).